iset.mm

```diff
@@ -145281,6 +145365,267 @@ mathematician Ptolemy (Claudius Ptolemaeus).  This particular version is
+    ( cc wcel caddc co c1 wceq w3a cpi c2 cdiv cmul cmin ccos cfv mulcl 3adant3
+    mpan2 eqtr3d csin halfpire recni coshalfpim syl adddir mp3an3 oveq1 mulid2i
+    3ad2ant1 syl6eq 3ad2ant3 wb subadd mp3an3an mpbird fveq2d ) ACDZBCDZABEFZGH
+    ZIZJKLFZAVCMFZNFZOPZVDUAPZBVCMFZOPURUSVFVGHZVAURVDCDZVIURVCCDZVJVCUBUCZAVCQ
+    SZVDUDUEUJVBVEVHOVBVEVHHZVDVHEFZVCHZVBUTVCMFZVOVCURUSVQVOHZVAURUSVKVRVLABVC
+    UFUGRVAURVQVCHUSVAVQGVCMFVCUTGVCMUHVCVLUIUKULTURUSVNVPUMZVAVKURVJUSVHCDZVSV
+    LVMUSVKVTVLBVCQSVCVDVHUNUORUPUQT $.
+
+  $( The sine and cosine of ` _pi / 4 ` .  (Contributed by Paul Chapman,
+     25-Jan-2008.) $)
+  sincos4thpi $p |- ( ( sin ` ( _pi / 4 ) ) = ( 1 / ( sqrt ` 2 ) ) /\
+                      ( cos ` ( _pi / 4 ) ) = ( 1 / ( sqrt ` 2 ) ) ) $=
+    ( cpi c4 cdiv co csin cfv c1 c2 csqrt wceq cmul cc wcel ax-mp recni 4re cc0
+    pire wbr clt ccos caddc halfcn ax-1cn 2halves sincosq1eq mp3an divmuldivapi
+    oveq2i 2cn 2ap0 mulid2i 2t2e4 oveq12i fveq2i recidapi oveq1i 2re redivclapi
+    eqtri mulassi 3eqtr3i sin2t sinhalfpi cr 4ap0 resincl remulcli 0le2 msqge0i
+    mulcli sqrtmulii sqrt1 3eqtr3ri cap wa wb cle sqrtcli sqrt2re 2pos gt0ap0ii
+    sqrtgt0ii pm3.2i divmulap2 mpbir 0re cioc 4pos divgt0ii 1re pigt2lt4 simpri
+    pipos ltdiv1ii dividapi breqtri ltleii cxr w3a 0xr elioc2 mpbir3an sin01gt0
+    mpbi mp2an sqrtmsqi eqtr2i ) ABCDZEFZGHIFZCDZJXIUAFZXLJXLXJXJKDZIFZXJXLXOJZ
+    GXKXOKDZJZHXNKDZIFGIFXQGXSGIHGHCDZAHCDZKDZEFZYCKDZKDHYCYBUAFZKDZKDZXSGYDYFH
+    KYCYEYCKXTLMZYHXTXTUBDGJZYCYEJUCUCGLMZYIUDGUENXTXTUFUGZUIUIYDXNHKYCXJYCXJKY
+    BXIEYBGAKDZHHKDZCDXIGHAHUDUJAROZUJUKUKUHYLAYMBCAYNULUMUNUTZUOZYPUNUIHYBKDZE
+    FZYAEFYGGYQYAEHXTKDZYAKDGYAKDYQYAYSGYAKHUJUKUPUQHXTYAUJUCYAAHRURUKUSOZVAYAY
+    TULVBUOYBLMYRYGJXTYAUCYTVKYBVCNVDVBVBUOHXNURXJXJXIVEMZXJVEMABRPVFUSZXIVGNZU
+    UCVHZVIXJUUCVJZVLVMVNYJXOLMXKLMZXKQVOSZVPXPXRVQUDXOQXNVRSXOVEMUUEXNUUDVSNOU
+    UFUUGXKVTOXKVTHURWAWCWBWDGXOXKWEUGWFZQXJVRSXOXJJQXJWGUUCXIQGWHDMZQXJTSUUIUU
+    AQXITSZXIGVRSZUUBABRPWNWIWJXIGUUBWKXIBBCDZGTABTSZXIUULTSHATSUUMWLWMABBRPPWI
+    WOXEBBPOVFWPWQWRQWSMGVEMUUIUUAUUJUUKWTVQXAWKQGXIXBXFXCXIXDNWRXJUUCXGNZXHXLX
+    JXMXLXOXJUUHUUNUTYCYEXJXMYKYPYBXIUAYOUOVBXHWD $.
+
+  $( The tangent of ` _pi / 4 ` .  (Contributed by Mario Carneiro,
+     5-Apr-2015.) $)
+  tan4thpi $p |- ( tan ` ( _pi / 4 ) ) = 1 $=
+    ( cpi c4 cdiv co ctan cfv csin c1 c2 cc wcel cc0 cap wceq mp2an sincos4thpi
+    wbr cr recni sqrt2re ccos cn pire 4nn nndivre simpri 2re sqrtgt0ii gt0ap0ii
+    csqrt 2pos recap0 eqbrtri tanvalap simpli oveq12i recclapi dividapi 3eqtri
+    ) ABCDZEFZUTGFZUTUAFZCDZHIUJFZCDZVFCDHUTJKVCLMQVAVDNUTARKBUBKUTRKUCUDABUEOS
+    VCVFLMVBVFNZVCVFNZPUFZVEJKVELMQVFLMQVETSZVETIUGUKUHUIZVEULOZUMUTUNOVBVFVCVF
+    CVGVHPUOVIUPVFVEVJVKUQVLURUS $.
+
+  $( The sine and cosine of ` _pi / 6 ` .  (Contributed by Paul Chapman,
+     25-Jan-2008.)  Replace OLD theorem.  (Revised by Wolf Lammen,
```

Comment:
"Replace OLD theorem." can/should be revoved (also in set.mm). I think the corresponding OLD theorem was removed already (a long time ago).

Reply:
Makes sense to me. Suggestion accepted.